action.yml

```
@@ -104,5 +106,10 @@ runs:
               "id": "snapshots",
               "username": "${{ env.JFROG_USER }}",
               "password": "${{ env.JFROG_TOKEN }}"
+            },
+            {
+              "id": "ossrh",
```

Repositories were defined with `ossrh-snapshots`
```suggestion
              "id": "ossrh-snapshots",
```